app/src/main/res/values/strings.xml

```diff
@@ -2579,4 +2579,12 @@ Please check the URL again."</string>
     <string name="text_to_search">Text to search</string>
     <string name="myprojects_list_menu_title_restore_projects">Restore Projects</string>
     <string name="sort_projects">Sort projects</string>
+    <string name="use_r8">Use R8</string>
+    <string name="activated">Activated</string>
+    <string name="obfuscate_def">Activating R8/ProGuard will minimize and obfuscate your project by renaming existing classes and methods on compilation time.</string>
+    <string name="code_shrinker_warning">Using R8/ProGuard might be risky to some extent. Make sure to provide a proper rules configuration file in order not to break your project.</string>
+    <string name="code_shrinker_rules">Code Shrinker Rules</string>
+    <string name="configure_shrinker">Configure R8/ProGuard by editing the rules file. This step is necessary because without a proper configuration, ProGuard might break your project.</string>
+    <string name="full_mode_def">Using this, you can configure specific local libs to be included in R8/ProGuarding process. (Note: Enabling this will ignore DEX files of the local libs you select and it will force Sketchware to compile them to dex, making compilation noticeably slower.)</string>
+    <string name="generate_debug">Generate debug files</string>
```

Comment:
there's an inconsistent string prefix; why not make all strings only used in Code Shrinking Manager have the prefix `code_shrinker_`?

Reply:
fixed